index.js

```diff
@@ -24,12 +24,31 @@ try {
   const logoPosition = core.getInput('logoPosition');
   const style        = core.getInput('style');
   const cacheSeconds = core.getInput('cacheSeconds');
+  const valColorRange = core.getInput('valColorRange');
+  const minColorRange = core.getInput('minColorRange');
+  const maxColorRange = core.getInput('maxColorRange');
+  const invertColorRange = core.getInput('invertColorRange');
 
   if (labelColor != '') {
     content.labelColor = labelColor;
   }
 
-  if (color != '') {
+  if (minColorRange != '' && maxColorRange != '' && valColorRange != '') {
+    var max = parseFloat(maxColorRange);
+    var min = parseFloat(minColorRange);
+    var val = parseFloat(valColorRange);
```

Comment:
I think these local variables can be `const`:

```suggestion
    const max = parseFloat(maxColorRange);
    const min = parseFloat(minColorRange);
    const val = parseFloat(valColorRange);
```

Reply:
Agree. I am still not a javascript programmer, so this kind of differences are still not obvious to me.